tests/integration/test_github.py

```diff
@@ -227,8 +227,6 @@ def test_update_pr_info(self):
         )
         pr_info = self.colin_project.get_pr_info(pr_id=1)
         assert pr_info.title == "changed"
-        assert pr_info.description == "changed description"
-
```

Comment:
Why this change?

Reply:
Just a mistake, I didn't notice I had deleted it.

Reply:
Otherwise LGTM, sorry for the misleading description of the issue.